pages/common/xh.md

```diff
@@ -0,0 +1,24 @@
+# xh
+
+> Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body (each key specified is added to the top-level JSON object e.g. `{"name": "john", "age": 25}`):
+
+`xh post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
+
+- Send a GET request with query parameters (e.g. `id=5&sort=true`):
+
+`xh get {{httpbin.org/get}} {{id==5}} {{sort==true}}`
```

Comment:
Is this a typo? Also, a small naming convention suggestion.

I've never used this command myself, so I don't know the syntax.
```suggestion
`xh get {{httpbin.org/get}} {{parameter_a=value_a}} {{parameter_b=value_b}}`
```

Reply:
No, that's not, `==` means that's query string
![image](https://user-images.githubusercontent.com/34353640/114308516-981a1800-9ae4-11eb-8fb1-797a96cf1784.png)